server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const cors = require('cors');
+
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+
+app.post('/api/login', (req, res) => {
+  const { email, password } = req.body;
+
+  // Here you would typically check the email and password against a database
+  // For simplicity, we will just check if they match a certain string
```

Comment:
This implementation is not secure. Never hardcode sensitive information such as usernames and passwords. Consider using a database or an environment variable.